src/amber/cli/commands/routes.cr

```diff
@@ -120,8 +128,7 @@ module Amber::CLI
             row.add_column route[l].to_s
           end
         end
-        puts table
-        exit
```

Comment:
Any reason to remove `exit`?

Reply:
Yes, there is no reason for the function named "print" to also exit automatically. I've moved "exit" to the run() method, where it properly exits after printing the table.